tests/FSharp.Compiler.ComponentTests/Language/MultiDimensionalArrayTests.fs

```diff
@@ -0,0 +1,52 @@
+﻿namespace FSharp.Compiler.ComponentTests.Language
+
+open Xunit
+open FSharp.Test.Compiler
+
+module MultiDimensionalArrayTests =
+
+    [<Theory>]
+    [<InlineData(1, "array2d")>]
+    [<InlineData(5, "array6d")>]
+    [<InlineData(31, "array32d")>]
+    let ``MultiDimensional array type can be written with or without backticks`` (commas: int, shortcut: string) =
```

Comment:
So I guess it's multidimensional, backtick, and shortcut - all are one word so no need for extra capitals ;)